index.html

```diff
@@ -879,7 +880,23 @@ <h4 class="media-heading">The first official news on the conference (June 1 2016
 		<li>Lightning talk (5min): A variation of the pecha kucha and ignite formats that we love at the useR! confereces: 15 slides shown for 20-20 seconds.</li>
 		<li>Poster: Abstracts accepted for posters will take place during an afternoon poster session, which is a social event. There are no parallel talks or events happening, so everyone can talk and stop by posters they are interested in. The dimensions of each poster should not exceed 4' x 4' or 120cm x 120cm.</li>
 	      </ul>
-	    
+
+	    <section id="it">
+	      <div class="page-header">
+		<h1>IT & AV Provision</h1>
+	      </div>
+	      <h2>Workshops</h2>
+	      <p>Presenters of workshops are advised that each workhop room is equipped with internet connection (Wi-Fi) and a large LCD screen (HD or FullHD resolution) with an HDMI connector. Presenters must bring their own laptops.</p>
+	      <h2>Oral Talks</h2>
+	      <p>Presenters are advised that each session room is equipped with:</p>
+	      <ul>
+	          <li>Internet connection (Wi-Fi)</li>
```

Comment:
@daroczig 
VPN: Maybe I will add a note in a further commit that presenters with special requests should contact the organizers per e-mail before the conference.
WiFi: Should be OK, but I will ask it from the IT guys. Anyway, the desktop PC is on a different network, it does not interfere at all with the load of the guest-WiFi (AFAIK).